src/plugins/intel_cpu/src/nodes/interpolate.cpp

```diff
@@ -43,6 +43,22 @@ using namespace Xbyak;
 
 namespace ov::intel_cpu::node {
 
+static inline bool isARM() {
+    bool isArm = false;
+#if defined(OPENVINO_ARCH_ARM) || defined(OPENVINO_ARCH_ARM64)
+    isArm = true;
+#endif
+    return isArm;
+}
+
+static inline bool isACL() {
+    bool isACL = false;
+#if defined(OV_CPU_WITH_ACL)
+    isACL = true;
+#endif
+    return isACL;
+}
+
```

Comment:
Don't think this is a good refactoring. We either should think about same approach inside whole plugin, or keep using already adopted ifdef structures.
Given the fact some parts of the code are placed under defines on purposes I would stick to the second one.

Reply:
@dmitry-gorokhov corrected